apollo-router/src/plugins/include_subgraph_errors.rs

```diff
@@ -42,36 +42,47 @@ impl Plugin for IncludeSubgraphErrors {
     }
 
     fn subgraph_service(&self, name: &str, service: subgraph::BoxService) -> subgraph::BoxService {
```

Comment:
`include_subgraph_errors.rs` is the best place I could find to add this functionality.